src/Parser/MarksmanshipHunter/CombatLogParser.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import SuggestionsTab from 'Main/SuggestionsTab';
+import Tab from 'Main/Tab';
+import Talents from 'Main/Talents';
+
+import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
+
+import CastEfficiency from './Modules/Features/CastEfficiency';
+
+
+import CooldownTracker from './Modules/Features/CooldownTracker';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+import DamageDone from './Modules/MMCore/DamageDone';
+
+class CombatLogParser extends CoreCombatLogParser {
+  static specModules = {
+    // MM Core
+    damageDone: DamageDone,
+
+    // Features
+    alwaysBeCasting: AlwaysBeCasting,
+    castEfficiency: CastEfficiency,
+    cooldownTracker: CooldownTracker,
+  };
+
+  generateResults() {
+    const results = super.generateResults();
+
+    this.selectedCombatant._combatantInfo.gear.forEach(function (value) {
+      const equippedItem = ITEMS[value.id];
+
+      if (equippedItem !== undefined && equippedItem.quality === 5) {
```

Comment:
The magic number `5` should be replaced with a constant, you can find item quality constants in the `common/ITEM_QUALITIES` file.